wp-content/plugins/core/src/Templates/Content/Panels/MediaText.php

```diff
@@ -0,0 +1,273 @@
+<?php
+
+
+namespace Tribe\Project\Templates\Content\Panels;
+
+use Tribe\Project\Panels\Types\MediaText as MediaTextPanel;
+use Tribe\Project\Templates\Components\Image;
+use Tribe\Project\Templates\Components\Content_Block;
+use Tribe\Project\Templates\Components\Text;
+use Tribe\Project\Templates\Components\Title;
+use Tribe\Project\Templates\Components\Button;
+use Tribe\Project\Templates\Components\Video;
+use Tribe\Project\Theme\Oembed_Filter;
+use Tribe\Project\Theme\Util;
+
+class MediaText extends Panel {
+
+	/**
+	 * Get the data.
+	 *
+	 * @return array
+	 */
+	public function get_data(): array {
+		$data       = parent::get_data();
+		$panel_data = $this->get_mapped_panel_data();
+		$data       = array_merge( $data, $panel_data );
+
+		return $data;
+	}
+
+	/**
+	 * Get the mapped panel data.
+	 *
+	 * @return array
+	 */
+	protected function get_mapped_panel_data(): array {
+		$data = [
+			'content_block'  => $this->get_media_text_content_block(),
+			'media'          => $this->get_media_text_media(),
+			'media_position' => $this->get_media_text_image_position(),
+			'layout'         => $this->get_media_text_layout(),
+		];
+
+		return $data;
+	}
+
+	/**
+	 * Overrides `get_classes()` from the Panel parent class.
+	 *
+	 * Return value is available in the twig template via the `classes` twig variable in the parent class.
+	 *
+	 * @return string
+	 */
+	protected function get_classes(): string {
+		$no_padding = '';
+		$wrapper_class = $this->get_media_text_wrapper_class();
+
+		if ( MediaTextPanel::OPTION_LAYOUT_FULL_BLEED === $this->panel_vars[ MediaTextPanel::FIELD_LAYOUT ] ) {
+			$no_padding = 's-wrapper--no-padding';
+		}
+
+		$classes = [
```

Comment:
Per our call last week, remember that we want to have parent wrapper modifier classes set around any options we have (image vs. video, left vs. right, full bleed vs. regular) for easy access.

Reply:
Seems like that'll cause bloat, in my opinion. The `s-wrapper` has a modifier` s-wrapper--no-padding` which already removes padding from the wrapper to handle the full bleed and contained experience. why would we do this on a panel by panel basis? The image position is different since it directly affects the padding of the content container within the panel itself.

Reply:
So last week we discussed how providing those "hooks" would be nice to have out of the box and give FE devs a variety of ways with which to quickly and efficiently target a given panel. I _think_ the pro of what it gives is greater than the con of a little bloat, but that's fair. Let's take a vote, good time to bring in the @frontend crew!

@frontend would it be of value to you to provide, out of the box, on the parent panel wrapper, a set of classes based on any options a panel has (thumb or down emoji)?